alts/src/generated/main/grpc/io/grpc/alts/internal/HandshakerServiceGrpc.java

```diff
@@ -104,6 +104,29 @@ public static abstract class HandshakerServiceImplBase implements io.grpc.Bindab
      * response before sending next request.
      * </pre>
      */
+    default io.grpc.stub.StreamObserver<io.grpc.alts.internal.HandshakerReq> doHandshake(
+        io.grpc.stub.StreamObserver<io.grpc.alts.internal.HandshakerResp> responseObserver) {
+      return io.grpc.stub.ServerCalls.asyncUnimplementedStreamingCall(getDoHandshakeMethod(), responseObserver);
+    }
+  }
+
+  /**
+   * Base class for the server implementation of the service HandshakerService
+   */
+  public static abstract class HandshakerServiceImplBase
+   implements io.grpc.BindableService, HandshakerServiceAsync {
+
+    /**
+     * <pre>
+     * Handshaker service accepts a stream of handshaker request, returning a
+     * stream of handshaker response. Client is expected to send exactly one
+     * message with either client_start or server_start followed by one or more
+     * messages with next. Each time client sends a request, the handshaker
+     * service expects to respond. Client does not have to wait for service's
+     * response before sending next request.
+     * </pre>
+     */
+    @java.lang.Override
     public io.grpc.stub.StreamObserver<io.grpc.alts.internal.HandshakerReq> doHandshake(
```

Comment:
Seems we don't need this method any more, as it is provided by the interface. I think we'd only need `bindService()` in the abstract class.

Reply:
Done